doc/pages/upgrade/020-upgrade_011_10.md

```diff
@@ -201,3 +201,15 @@ kubectl edit astarte -n astarte
 Find the `version` field in the Astarte Spec section and change it according to your needs. Once the
 yaml file will be saved, the Operator will take over ensuring the reconciliation of your Astarte
 instance to the requested version.
+
+### Caveats
+If Astarte v0.11 employs a persistent volume to store CA certificate and private key, upgrading to
```

Comment:
Remove `If`, it gives the idea that this is true only sometimes while it's always true.
"Astarte v0.11 employs [...], while upgrading to v1.0 [...]"